cms/envs/acceptance.py

```diff
@@ -80,6 +80,14 @@ def seed():
             'timeout': 30,
         },
         'ATOMIC_REQUESTS': True,
+    },
+    'student_module_history': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': TEST_ROOT / "db" / "test_student_module_history.db",
+        'TEST_NAME': TEST_ROOT / "db" / "test_student_module_history.db",
+        'OPTIONS': {
+            'timeout': 30,
+        },
```

Comment:
Django doesn't do cross-database transactions, so even having it on doesn't guarantee that a CSMHE write will success during a CSM update that is happening in the main edx database.  Given that lack of atomicity, it didn't seem to be worthwhile (several of us came to this conclusion independently).